R/init.R

```diff
@@ -95,17 +99,15 @@
 init <- function(data,
                  modules,
                  filter = teal_slices(),
-                 title = build_app_title(),
-                 header = tags$p(),
-                 footer = tags$p(),
-                 id = lifecycle::deprecated(),
-                 landing_popup = NULL) {
+                 title = lifecycle::deprecated(),
```

Comment:
init's example still contains title, header and footer.

Reply:
🤦🏽 Removed it now